specifyweb/frontend/js_src/lib/localization/workbench.ts

```diff
@@ -383,12 +383,7 @@ export const wbText = createDictionary({
     `,
   },
   startUpload: {
-    'en-us': 'Begin Data Set Upload?',
-    'ru-ru': 'Начать загрузку набора данных?',
-    'es-es': '¿Comenzar carga de conjunto de datos?',
-    'fr-fr': "Commencer le téléchargement de l'ensemble de données ?",
-    'uk-ua': 'Почати завантаження набору даних?',
-    'de-ch': 'Hochladen des Datensatzes beginnen?',
+    'en-us': 'Begin Data Set {type:string}?',
```

Comment:
while I like the dry-ness of this solution, I am beginning to realize that being too smart with reusing localization strings is not great:
- it makes autogenerated Google Translate translations worse
- it doesn't give translators context about what the "type" should be, thus leaving them unable to translate this correctly
- some languages might not be flexible enough to accommodate this pattern

is "type" here just one of 2 possible values? if so I would suggest we have separate strings for these

Reply:
Agreed, it was just too convenient to reuse them.